.github/workflows/tests.yml

```diff
@@ -14,91 +14,24 @@ on:  # yamllint disable-line rule:truthy
 
 jobs:
   changelog:
-    uses: ansible-network/github_actions/.github/workflows/changelog.yml@main
+    uses: ansible/ansible-content-actions/.github/workflows/changelog.yaml@main
```

Comment:
We need to add a tox-ansible.ini [ref](https://github.com/ansible-collections/cisco.ios/blob/main/tox-ansible.ini)
For the metric to form.

Reply:
Note - the file can be left blank as tox-ansible takes care of the metric parts on its own. But I guess the file needs to be there.